src/pages/black-friday.tsx

```diff
@@ -38,72 +54,109 @@ const BlackFriday = ({ lang, metatagsDescriptions, langJson, navbarLang, footerL
       description: langJson.blackFriday.FeatureSection.feature4.subtitle1,
     },
   ];
+
+  const onBusinessPlansSelected = (isBusiness) => {
+    setIsBusiness(isBusiness);
+  };
+
+  const onCheckoutButtonClicked = (priceId, isCheckoutForLifetime) => {
+    const lifetimeSpacePlan = products?.individuals[Interval.Lifetime].find((product) => product.priceId === priceId);
+
+    const couponCodeForCheckout = individualCoupon?.name;
+
+    const planType = isBusiness ? 'business' : 'individual';
+    stripeService.redirectToCheckout(priceId, currencyValue, planType, isCheckoutForLifetime, couponCodeForCheckout);
+  };
+
   return (
     <Layout title={metatags[0].title} description={metatags[0].description} segmentName="Black Friday">
-     
       <Navbar
         lang={lang}
         darkMode={true}
-        isBlackfriday={true}
         textContent={navbarLang}
         cta={['Hide Login']}
         isLinksHidden
-        
+        hideNavbar
+        fixed
+        isBlackfriday
       />
 
-
       <HeroSection lang={lang} textContent={langJson.blackFriday} />
 
-      <BestStorageSection textContent={langJson.blackFriday} lang={lang} />
-
-      <SuiteSection lang={lang} textContent={langJson.blackFriday} />
-
-      <CtaSection textContent={langJson.cta} lang={lang} />
-
+      <PricingSectionWrapper
+        textContent={pricingLang.tableSection}
+        lang={lang}
+        products={products}
+        loadingCards={loadingCards}
+        onBusinessPlansSelected={onBusinessPlansSelected}
+        onCheckoutButtonClicked={onCheckoutButtonClicked}
+        lifetimeCoupons={lifetimeCoupons}
+        hideFreeCard
+        darkMode
+        backgroundColorComponent="bg-highlight"
+        hideTitle
+        decimalDiscount={{
+          individuals: individualCoupon?.percentOff && 100 - individualCoupon?.percentOff,
```

Comment:
Extract the operation to a constant and use it for individuals, lifetime and business. Something like: `const decimalDiscount = individualCoupon?.percentOff && 100 - individualCoupon?.percentOff,`

Reply:
Okay